src/AzureSignTool/AzureSignTool.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
I don’t think multi targeting works for global tools and “dotnet pack” will fail. I’m fine dropping netcoreapp2.1 entirely.

Reply:
https://github.com/dotnet/sdk/issues/9917 For some detail.

Reply:
I’m fine dropping netcoreapp2.1 entirely too 👍